src/popup.js

```diff
@@ -0,0 +1,206 @@
+xBtn.innerHTML = 'X';
+popupHeader.appendChild(xBtn);
+
+// popup-image
+const popupImage = document.createElement('img');
+popupImage.id = 'popupImage';
+popupImage.classList = 'popup-image';
+popupImage.src = pokemon3; /// ////////// for test
+popupImage.alt = 'pokemon';
+popupWindow.appendChild(popupImage);
+
+// popup-image-details
+const popupDetail = document.createElement('div');
+popupDetail.id = 'popupDetail';
+popupDetail.classList = 'popup-image-detail d-grid-2col';
+/// /////////////// for test
+popupDetail.innerHTML = `
+                <p class="popup-detail-item">Color : Green</p>
+                <p class="popup-detail-item">Age : 6 years</p>
+                <p class="popup-detail-item">Power : 132</p>
+                <p class="popup-detail-item">Speed : 42 KM/h</p>
+`;
+popupWindow.appendChild(popupDetail);
+
+// popup-comment-header
+const popupCommentHeader = document.createElement('h3');
+popupCommentHeader.classList = 'popup-comment-header';
+popupCommentHeader.innerHTML = 'Comments (4)'; /// /////// for test
+popupWindow.appendChild(popupCommentHeader);
+
+// popup-engage section
+const engage = document.createElement('div');
+engage.classList = 'd-grid-2col';
+popupWindow.appendChild(engage);
+
+// popup-comment-review
+const popupCommentReview = document.createElement('div');
+popupCommentReview.classList = 'popup-comment-review';
+engage.appendChild(popupCommentReview);
+
+// popup-comment-list
+const popupCommentList = document.createElement('ul');
+popupCommentList.id = 'popupCommentList';
+popupCommentList.classList = 'popup-comment-list';
+popupCommentReview.appendChild(popupCommentList);
+
+// popup-new-comment
+const popupNewComment = document.createElement('div');
+popupNewComment.classList = 'popup-new-comment';
+engage.appendChild(popupNewComment);
+
+// popup-comment-input name
+const popupCommentInputName = document.createElement('input');
+popupCommentInputName.id = 'popupCommentInputName';
+popupCommentInputName.classList = 'popup-comment-input';
+popupCommentInputName.type = 'text';
+popupCommentInputName.placeholder = 'Your name';
+popupNewComment.appendChild(popupCommentInputName);
+
+// popup-comment-input comment
+const popupCommentInputComment = document.createElement('textarea');
+popupCommentInputComment.id = 'popupCommentInputComment';
+popupCommentInputComment.classList = 'popup-comment-input popup-new-comment-text';
+popupCommentInputComment.cols = '15';
+popupCommentInputComment.rows = '4';
+popupCommentInputComment.placeholder = 'Comment...';
+popupNewComment.appendChild(popupCommentInputComment);
+
+// popup-comment-btn
+export const popupCommentBtn = document.createElement('a');
+popupCommentBtn.id = 'popupCommentBtn';
+popupCommentBtn.classList = 'popup-comment-btn';
+popupCommentBtn.innerHTML = 'Comment';
+popupNewComment.appendChild(popupCommentBtn);
+
+xBtn.addEventListener('click', () => {
+  popup.classList.remove('display');
+  popupCommentList.innerHTML = '';
+});
```

Comment:
- [ ] [Optional] Since the height of the popup is larger than the view height, it is better to add a function where users can click outside the popup to close it.